### DISCUSSION TITLE
Radio Group - vertical gone in v6?

### DISCUSSION THREAD

**Discussion by atumas-bananamilk** [mantinedev/mantine#4250] [Feedback]
Tons of options have disappeared from `Radio.Group` in `6.0`.
Why? We were using them.

v5:
![Screenshot 2023-05-15 at 13 25 07](https://github.com/mantinedev/mantine/assets/22228431/9d0685bf-2249-4f10-96b4-604232fcff58)

v6:
![Screenshot 2023-05-15 at 13 25 12](https://github.com/mantinedev/mantine/assets/22228431/c90c1204-3a76-41d4-980a-842bd358109e)

**Comment by rtivital** [comment]
Use Stack component.
